src/opera/test/pge/dist_s1/test_dist_s1_pge.py

```diff
@@ -74,6 +130,8 @@ def test_dist_s1_pge_execution(self):
         """
         runconfig_path = join(self.data_dir, 'test_dist_s1_config.yaml')
 
+        DistS1PgeTestCase._create_dummy_input_files(runconfig_path)
```

Comment:
I think we could make this call part of `setUp` so we don't have to remember to call it in every test case that needs it (which is likely all of them anyhow)